tests/Integration/SecurityTest.php

```diff
@@ -130,10 +135,20 @@ private function loadSecurityConfiguration()
      */
     protected function setUserRole(string $role)
     {
+        if (class_exists(InMemoryUser::class)) {
+            $user = new InMemoryUser('test', 'test', [$role]);
+            $token = \Mockery::mock(TokenInterface::class);
+            $token->shouldReceive('getUser')->andReturn($user);
+            $token->shouldReceive('getRoleNames')->andReturn([$role]);
+            if (method_exists(TokenInterface::class, 'isAuthenticated')) {
+                $token->shouldReceive('isAuthenticated')->andReturn(true);
+            }
+        } else {
+            $token = new AnonymousToken('test', 'anon', [$role]);
+        }
```

Comment:
Sorry about that ugly compatibility layer but it's really needed as `AnonymousToken` does not exist anymore with the new security system.

Reply:
No worries, thanks for doing the dirty work in the first place! :heart: